pro/src/components/ArchiveConfirmationModal/ArchiveConfirmationModal.tsx

```diff
@@ -9,20 +10,30 @@ interface OfferEducationalModalProps {
   onDismiss(): void
   onValidate(): void
   hasMultipleOffers?: boolean
+  selectedOffers?: CollectiveOfferResponseModel[]
 }
 
 export const ArchiveConfirmationModal = ({
   onDismiss,
   onValidate,
   hasMultipleOffers = false,
+  selectedOffers = [],
 }: OfferEducationalModalProps): JSX.Element => {
   const location = useLocation()
   const { logEvent } = useAnalytics()
 
   function onConfirmArchive() {
+    const collectiveOfferIds = []
+
+    for (const offer of selectedOffers) {
+      collectiveOfferIds.push(offer.id.toString())
+    }
+
```

Comment:
Pas important, mais ça fait des lignes en moins
```suggestion
    const collectiveOfferIds = selectedOffers.map((offer) => offer.id.toString())
```